.github/workflows/canary-integration-test.yml

```diff
@@ -948,6 +948,7 @@ jobs:
         yq w -i -d1 cluster-test.yaml spec.dashboard.enabled false
         yq w -i -d1 cluster-test.yaml spec.storage.useAllDevices false
         yq w -i -d1 cluster-test.yaml spec.storage.deviceFilter ${BLOCK}1
+        yq w -i -d1 cluster-test.yaml spec.cephVersion.image ceph/daemon-base:latest-pacific-devel
```

Comment:
ceph/ceph:v16.2.2 may be out soon anyway, maybe that will contain the fix?

Reply:
The backport merged yesterday and the branch locked the same day I'm not sure if it made it or not...

Reply:
Also, they are still a few cephfs-mirror I'm looking forward to testing so I believe we will stick with `latest-pacific-devel` for a few weeks.